components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -991,6 +991,7 @@ func getExistingAndRequestedConfigForPostgres(args []string, infra *AutomateHAIn
 	//Getting Existing config from server
 	var existingConfig PostgresqlConfig
 	var reqConfig PostgresqlConfig
+	var dummyConfig PostgresqlConfig
```

Comment:
it can be named better, something  like blankConfig or emptyConfig

Reply:
Done